dask_glm/tests/test_utils.py

```diff
@@ -70,9 +71,33 @@ def test_add_intercept_dask():
     assert_eq(result, expected)
 
 
+def test_add_intercept_sparse():
+    from sparse._utils import assert_eq
```

Comment:
@hameerabbasi is there a public API to assert that two sparse arrays are equal?

Reply:
Is all you're using `COO`? In general, for future compat as well, I'd prefer you used `(a == b).all()`.

Reply:
Looks like we're supporting DOK too. But I'm not sure whether we care about the exact output format (maybe @jdlesage has thoughts).

Reply:
I can add some support to `DOK` for NumPy operators and then that'd be all that's needed.

Reply:
`DOK` is very primitive, I'd recommend not supporting it for now.